tests/feature_extraction/test_feature_calculations.py

```diff
@@ -195,6 +195,14 @@ def test_augmented_dickey_fuller(self):
         self.assertLessEqual(res["attr_pvalue"], 0.05)
         self.assertEqual(res["attr_usedlag"], 0)
 
+        # Check if LinAlgError and ValueError are catched
```

Comment:
http://www.voidspace.org.uk/python/mock/patch.html

you patch the function, change it to one that raises the error of interest, always, no madder which input